--- a/openjdk/hotspot/src/share/vm/memory/dump.cpp   2019-11-30 18:24:36.750323337 +0000
+++ b/openjdk/hotspot/src/share/vm/memory/dump.cpp   2019-11-30 18:24:51.150696251 +0000
@@ -1553,7 +1553,7 @@
     if (computed_jsum != file_jsum) {
       tty->cr();
       tty->print_cr("Preload failed: checksum of class list was incorrect.");
-      exit(1);
+      // exit(1);
     }
 
     tty->print_cr("done. ");
